torch/_inductor/utils.py

```diff
@@ -373,10 +373,10 @@ def is_pointwise_use(
     Uses in views ops will follow the views uses
     """
 
-    if not use.op == "call_function":
-        return False
     if not (
-        isinstance(use.target, torch._ops.OpOverload) or use.target is operator.getitem
+        use.op == "call_function"
+        and isinstance(use.target, torch._ops.OpOverload)
+        or use.target is operator.getitem
     ):
```

Comment:
To me, this reads less clearly now. if you're going to combine booleans can you use parentheses ?

Reply:
I think in general combining conditionals like this is harder to parse and also makes it harder to insert breakpoints